vermouth/tests/integration_tests/test_integration.py

```diff
@@ -130,7 +130,9 @@ def _interaction_equal(interaction1, interaction2):
     ['tier-0', 'mini-protein3_trp-cage'],
     ['tier-1', 'bpti'],
     ['tier-1', 'lysozyme'],
+    ['tier-1', 'lysozyme_prot'],
```

Comment:
What's the difference between lysosyme_prot and lysozyme? Could you put that in the README of the _prot testcase?

Reply:
added README; FYI this one tests the different protonation states for the charmm AAs